src/is.test.ts

```diff
@@ -46,3 +47,97 @@ describe('isArray(value)', () => {
     expect(isArray(1)).to.equal(false);
   });
 });
+
+// isBoolean(value)
+describe('isBoolean(value)', () => {
+  it('Should be a function', () => {
+    expect(isBoolean).to.be.a('function');
+  });
+
+  it('Should return true when "value" is type boolean', () => {
+    expect(isBoolean(true)).to.equal(true);
+  });
+
+  it('Should return false when "value" is not type boolean', () => {
+    expect(isBoolean('This is a string')).to.equal(false);
+  });
+});
+
+// isInfinity(value)
+describe('isInfinity(value)', () => {
+  it('Should be a function', () => {
+    expect(isInfinity).to.be.a('function');
+  });
+
+  it('Should return true when "value" is Infinity', () => {
+    expect(isInfinity(Infinity)).to.equal(true);
+  });
+
+  it('Should return false when "value" is not Infinity', () => {
+    expect(isInfinity('This is a string')).to.equal(false);
+  });
+});
+
+// isInteger(value)
+describe('isInteger(value)', () => {
+  it('Should be a function', () => {
+    expect(isInteger).to.be.a('function');
+  });
+
+  it('Should return true when "value" is an Integer', () => {
+    expect(isInteger(24)).to.equal(true);
+  });
+
+  it('Should return false when "value" is not an Integer', () => {
+    expect(isInteger(24.245)).to.equal(false);
+  });
+});
+
+// isIntegerArray(value)
+describe('isInteger(value)', () => {
```

Comment:
Be careful, when copying and pasting code, to check each line.

```suggestion
describe('isIntegerArray(value)', () => {
```